src/test/regress/multi_schedule

```diff
@@ -17,6 +17,7 @@
 # ---
 test: multi_extension
 test: multi_table_ddl
+test: multi_schema_support
```

Comment:
Shall we add this new test to the end of the file? I think "Tests around schema changes" implies the "table schema" here.

Similarly in multi_task_tracker_extra_schedule.